build.psm1

```diff
@@ -3,7 +3,7 @@
 $script:TestModulePathSeparator = [System.IO.Path]::PathSeparator
 
 $dotnetCLIChannel = "release"
-$dotnetCLIRequiredVersion = "2.0.2"
+$dotnetCLIRequiredVersion = $(Get-Content $PSScriptRoot/global.json | ConvertFrom-Json).Sdk.Version
```

Comment:
Should we stop if global.json does not contain Sdk version?

Reply:
It would be highly unusual for global.json to be missing that field since that's about its only purpose.